### PR TITLE
cleanup formatting and credits

### PR DESCRIPTION
**Pull request by Moshe-Immerman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -327,38 +327,38 @@ Services to securely store your Docker images.
...
-* [Marathon](https://mesosphere.github.io/marathon/docs/) - Marathon is a private PaaS built on Mesos. It automatically handles hardware or software failures and ensures that an app is "always on")
-* [MCollective Docker Agent](https://github.com/m4ce/mcollective-docker-agent) - Uses MCollective to orchestrate your Docker containers and images [@m4ce](https://github.com/m4ce)
-* [Nomad Project](https://www.nomadproject.io/) - Easily deploy applications at any scale. A Distributed, Highly Available, Datacenter-Aware Scheduler.
+* [Mantl](https://github.com/mantl/mantl) - Mantl is a modern platform for rapidly deploying globally distributed services
+* [Marathon](https://mesosphere.github.io/marathon/docs/) - Marathon is a private PaaS built on Mesos. It automatically handles hardware or software failures and ensures that an app is "always on"
+* [MCollective Docker Agent](https://github.com/m4ce/mcollective-docker-agent) - Uses MCollective to orchestrate your Docker containers and images by  [@m4ce](https://github.com/m4ce)
+* [Nomad](https://www.nomadproject.io/) - Easily deploy applications at any scale. A Distributed, Highly Available, Datacenter-Aware Scheduler by @hashicorp
```
Comment (veggiemonk) [inline README.md]: Should use github URL: https://github.com/hashicorp/nomad

**Comment by veggiemonk** [comment]
Hi  @Moshe-Immerman 
You did a good job on this PR! Would you be interested in becoming a maintainer?

**Comment by Moshe-Immerman** [comment]
@veggiemonk fixed, and sure would love to help out as a maintainer

**Comment by veggiemonk** [comment]
@Moshe-Immerman  Done ! And Welcome to the awesome-docker team!

Don't forget to read this short intro for new maintainers: https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md#new-collaborators
You can also add your twitter account in that file `CONTRIBUTING.md` if you wish

You can always ask question in the gitter chat room if you have any!
Don't hesitate to share ideas and give feedback. It is always appreciated.

**Pull request merged**